dart/chatbot-openai/server/genezio.yaml

```
@@ -1,16 +1,52 @@
+# The name of the project.
 name: chat-with-chatgpt
+# The region where the project is deployed.
 region: us-east-1
-language: dart 
-cloudProvider: genezio
-sdk:
-  language: dart
-  path: ../client/lib/sdk
-scripts:
-  preFrontendDeploy: cd ../client/ && flutter clean && flutter pub get && flutter build web
+# The version of the Genezio YAML configuration to parse.
+yamlVersion: 2
+backend:
+  # The root directory of the backend.
+  path: .
+  # Information about the backend's programming language.
+  language:
+    # The name of the programming language.
+    name: dart
+  # Classes that will be deployed
+  classes:
+    - name: ChatBackend
+      path: ./lib/chat_backend.dart
+      type: jsonrpc
+  # Scripts are running in the specified `path` directory.
+  scripts:
+    # List of scripts to run before deploying the backend.
+    deploy:
+      - dart pub get
+    # List of scripts to run before starting `genezio local`.
+    local:
+      - dart pub get
+  # The cloud provider where the backend is deployed.
+  cloudProvider: genezio
+# Information about the frontend, including the path, language, and publish directory.
+# It is optional. It can also be an array if there are multiple frontends you want to deploy.
 frontend:
-  path: ../client/build/web
-classes:
-  - path: lib/chat_backend.dart
-    type: jsonrpc
-    name: ChatBackend
-    methods: []
+  # The folder where the frontend scripts will run.
+  path: ../client/
```

You should move `genezio.yaml` in the root. We want to encourage the "workspace" setup.
```suggestion
  path: client
```

---

I tried to move `genezio.yaml` to workspaces and that yield a bunch of errors. 
I didn't continue to investigate the problem. For now, we are gonna put `genezio.yaml` in the backend directory.